frontend/tickers.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/grpcoin/grpcoin/gdax"
+)
+
+type tickerService struct {}
+
+var newTicker tickerService
+
+func wsTickers(w http.ResponseWriter, r *http.Request) {
+	conn, err := upgrader.Upgrade(w, r, nil)
+	defer conn.Close()
+	if err != nil {
+		log.Println(err)
+		return
+	}
+
+	client := &Client{hub: hub, conn: conn, send: make(chan []byte, 256)}
+	client.hub.register <- client
+
+	ctx, stop := context.WithTimeout(context.Background(), time.Hour*5)
```

Comment:
this is not useful, Cloud Run's max request limit is 1h, so we need reconnect logic in the browser.